lighthouse-core/fraggle-rock/gather/navigation-runner.js

```diff
@@ -287,7 +287,8 @@ async function navigation(options) {
     skipAboutBlank: configContext.skipAboutBlank,
   };
 
-  return Runner.run(
+  const runner = new Runner(config, computedCache);
+  await runner.gatherPhase(
```

Comment:
We already use phase for the old before/afterPass and the new start/stopInstrumentation, so ideally we wouldn't overload it. gather and audit are already verbs, so the function names could be just those :)

Reply:
Name change SGTM